app/presenters/BasePresenter.php

```diff
@@ -3,7 +3,7 @@
 namespace App;
 
 use Nette;
-use WebLoader;
+//use WebLoader;
```

Comment:
Proč nahrazovat webloader za include jednotlivých souborů? to je cesta zpět za mě

Reply:
Protože WebLoader mel posledni commit před několika lety a neni kompatibilni s Nette 3.0.

Reply:
u hospodareni mame webpack nebo pak fmasa jeste doporucoval gulp